README.md

```diff
@@ -68,6 +68,25 @@ Followed by running your process with the appropriate ENV.
 $ manifold run ./bin/server
 ```
 
+## OAuth Identity
+
+Manifold gives you the ability to login to using a third-party OAuth provider such as GitHub.
+
+### GitHub Authentication
```

Comment:
Now that I'm seeing this here, I'm thinking it's going to be confusing to the end-user if they're looking to use Github auth. They might think they need to supply these, rather than this just being information for development of the CLI.

What do packages normally do for "development" guidelines? separate file from the README?

Reply:
It might make sense to put it into the Marketplace docs, where the Stripe configuration is there, since the Marketplace will need the ClientID and ClientSecret as well.

Reply:
[Added to the README.md in marketplace](https://github.com/manifoldco/marketplace/blob/f11430189824a21b49728b285ee2cdf1b4958c52/README.md#github-authentication).

Reply:
Since you moved it, will you be taking this out of here? So to not confused the general public?

Reply:
Yeah, all the README.md changes should be gone